lib/functions/compilation/patch/drivers_network.sh

```diff
@@ -672,28 +672,3 @@ driver_rtl8723cs() {
 	fi
 
 }
-
-patch_drivers_network() {
-	display_alert "Patching network related drivers"
-
-	driver_generic_bring_back_ipx
-	driver_rtl8152_rtl8153
-	driver_rtl8189ES
-	driver_rtl8189FS
-	driver_rtl8192EU
-	driver_rtl8811_rtl8812_rtl8814_rtl8821
-	driver_xradio_xr819
-	driver_rtl8811CU_rtl8821C
-	driver_rtl8188EU_rtl8188ETV
-	driver_rtl88x2bu
-	driver_rtw88
-	driver_rtl88x2cs
-	driver_rtl8822cs_bt
-	driver_rtl8723DS
-	driver_rtl8723DU
-	driver_rtl8822BS
-	driver_uwe5622_allwinner
-	driver_rtl8723cs
```

Comment:
- are all those drivers mainlined and do we know drivers in mainline works better?
- xradio is needed for most popular allwinner board (there is a solution for 6.1+ ...)
- 8189es and fs are present on many boards and tv boxes

Reply:
Let me rephrase this.

We had 2 code paths before this PR:

1) The actually used path. In `drivers-harness.sh`, function `kernel_drivers_create_patches()` called `kernel_drivers_prepare_harness()` which in turn has an array of function names, defined in `drivers_network.sh` (for example, `driver_generic_bring_back_ipx()`).

2) The old, unused, uncalled path, which was dead code: `kernel-drivers.sh` had a function `prepare_extra_kernel_drivers()` which called `patch_drivers_network()` in `drivers_network.sh`. This path has not been used since armbian-next and is being removed here.

Item 2 is what is being removed, thus removing code that was dead for 6 months or more.
No actual changes are expected, except to get rid of junk.